dpctl/_sycl_queue_manager.pyx

```diff
@@ -210,6 +210,14 @@ cpdef get_current_backend():
     return _mgr.get_current_backend()
 
 
+def _get_registered_context_manager(sycl_queue):
+    try:
+        from numba_dppy import get_context_manager
+        return get_context_manager(sycl_queue)
+    except:
```

Comment:
I am surprised flake, or other linters did not complain. It should be `except Exception:` or some such.

Reply:
My local PyLance linter in VSCode points it out.

Reply:
```suggestion
    except Exception:
```